### PR TITLE
Update 'Putting it all together' link in off-canvas docs - Issue 10327

### PR DESCRIPTION
**Pull request by codetheorist**
As-per [issue 10327](https://github.com/zurb/foundation-sites/issues/10327), I have updated the link under 'Putting it all together' heading to [http://foundation.zurb.com/building-blocks/blocks/multilevel-offcanvas-menu.html](http://foundation.zurb.com/building-blocks/blocks/multilevel-offcanvas-menu.html).

@IamManchanda Here's the afore-mentioner PR.

### CODE REVIEW

File: docs/pages/off-canvas.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ If you need a simple bar to contain your hamburger icon/s and toggle the off-can
 
 #### Responsive Off-Canvas (Putting it all together)
 
-For an example of off-canvas on small screens and Top Bar Menu with Dropdowns, check out this Building Block: http://zurb.com/building-blocks/top-bar-with-off-canvas
+For an example of off-canvas on small screens and Top Bar Menu with Dropdowns, check out this Building Block: http://foundation.zurb.com/building-blocks/blocks/multilevel-offcanvas-menu.html
```
Comment (IamManchanda) [inline docs/pages/off-canvas.md]: Write 
```
#### Off-Canvas (Putting it all together)

For an example of off-canvas, checkout this top bar with off-canvas navigation and dropdowns for submenus building block.
```

**Review by IamManchanda — COMMENTED** [review]
@codetheorist Edit this

**Comment by codetheorist** [comment]
@IamManchanda I have updated the link to the link provided by yourself.

**Review by IamManchanda — APPROVED** [review]
LGTM, but 
Hesitant to merge by myself... cc @kball

**Review by codetheorist — COMMENTED** [review]
Changes requested by @IamManchanda in issue

**Review by codetheorist — COMMENTED** [review]
Changes as-per those requested by @IamManchanda

**Comment by kball** [comment]
Merged, thanks!

**Pull request merged**